src/style.css

```diff
@@ -1 +1,81 @@
-/* add styles here */
+@font-face {
+  font-family: 'Avenir Book';
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+}
+
+body {
+  font-family: 'Avenir Book', sans-serif;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  display: block;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  box-sizing: border-box;
+}
+
+.search-bar__big {
+  height: 70px;
+  font-size: 16px;
+  padding-left: 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: left 26px center;
+  background-color: #fff;
+}
+
+.search-bar__small {
+  height: 42px;
```

Comment:
Similar to the 'search-bar__big' class, you have also fixed the height of the 'search-bar__small' class. It is recommended not to fix the container size unless there is a requirement.